plugins/modules/zos_find.py

```diff
@@ -218,6 +207,14 @@
 
 
 EXAMPLES = r"""
+- name: Exclude all members starting with characters 'TE' in a given list datasets patterns
+  zos_find:
+    excludes: '(^te.*)'
```

Comment:
Could you also add an example on how to exclude members but also having the data set name? I'm afraid only having this example would scare users into thinking that you can only exclude data set names or member names at a time

Reply:
Thank you!